examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -23,6 +25,8 @@
 raw_fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
 event_fname = data_path + '/MEG/sample/sample_audvis_raw-eve.fif'
 
+plt.ion()
+
```

Comment:
The plots weren't working for me w/o that, but it may have been a backend problem on my computer. Either way I'm gonna remove it before I push a final commit